finish/pom.xml

```diff
@@ -48,43 +60,41 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>javax.ws.rs</groupId>
-            <artifactId>javax.ws.rs-api</artifactId>
-            <version>2.0.1</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>com.ibm.websphere.appserver.api</groupId>
-            <artifactId>com.ibm.websphere.appserver.api.jaxrs20</artifactId>
-            <version>1.0.10</version>
+            <groupId>io.openliberty.features</groupId>
+            <artifactId>jaxrs-2.0</artifactId>
+            <type>esa</type>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>javax.json</groupId>
-            <artifactId>javax.json-api</artifactId>
-            <version>1.0</version>
+            <groupId>io.openliberty.features</groupId>
+            <artifactId>jsonp-1.0</artifactId>
+            <type>esa</type>
             <scope>provided</scope>
         </dependency>
-	<dependency>
-	    <groupId>javax.xml.bind</groupId>
-	    <artifactId>jaxb-api</artifactId>
-	    <version>2.2.11</version>
-	</dependency>
-	<dependency>
-	    <groupId>com.sun.xml.bind</groupId>
-	    <artifactId>jaxb-core</artifactId>
-	    <version>2.2.11</version>
-	</dependency>
-	<dependency>
-	    <groupId>com.sun.xml.bind</groupId>
-	    <artifactId>jaxb-impl</artifactId>
-	    <version>2.2.11</version>
-	</dependency>
-	<dependency>
-	    <groupId>javax.activation</groupId>
-	    <artifactId>activation</artifactId>
-	    <version>1.1.1</version>
-	</dependency>
+    	<dependency>
+    	    <groupId>javax.xml.bind</groupId>
+    	    <artifactId>jaxb-api</artifactId>
+    	    <version>2.2.11</version>
+            <scope>test</scope>
+    	</dependency>
+    	<dependency>
+    	    <groupId>com.sun.xml.bind</groupId>
+    	    <artifactId>jaxb-core</artifactId>
+    	    <version>2.2.11</version>
+            <scope>test</scope>
+    	</dependency>
+    	<dependency>
+    	    <groupId>com.sun.xml.bind</groupId>
+    	    <artifactId>jaxb-impl</artifactId>
+    	    <version>2.2.11</version>
+            <scope>test</scope>
+    	</dependency>
+    	<dependency>
+    	    <groupId>javax.activation</groupId>
+    	    <artifactId>activation</artifactId>
+    	    <version>1.1.1</version>
+            <scope>test</scope>
```

Comment:
indentation is incorrect.

Reply:
The code in Anshul's local directory looks fine, and he is not able to commit the code again since the git indicate that there's no change

Reply:
I double checked the code and there seems is a format issue for the whole area, please try modifying the whole dependency using 4 spaces instead of just the scope line,
![image](https://user-images.githubusercontent.com/32473795/45177405-d50d2800-b1e0-11e8-8828-60de9327ad47.png)